### PR TITLE
 Merge latest strings from activity-stream changeset a772240e93f97e05886ca7b3d2e7ce58deb7ad0d

### PR DESCRIPTION
**Pull request by Mardak**
Looks like we'll try to land some more stuff before our string / feature freeze after all.

Downloads in Highlights https://github.com/mozilla/activity-stream/pull/4076
screen shot 2018-04-16 at 4 47 02 pm

Highlights options https://github.com/mozilla/activity-stream/pull/4111
screen shot 2018-04-20 at 1 53 42 pm

Removing provider from sponsored option https://github.com/mozilla/activity-stream/pull/4108
screen shot 2018-04-18 at 3 23 40 pm

~~Notably, the highlights options for about:preferences has not landed yet, but probably will today/Friday, and design has confirmed the strings~~ https://bugzilla.mozilla.org/show_bug.cgi?id=1432595#c11 Landed!

### CODE REVIEW

File: l10n/templates/strings.properties
Comment (flodolo) [inline l10n/templates/strings.properties]: This is the only string I'm not sure about. Is "Saves" something that shows up in Pocket's UI, or just a generic short term for "saved items"? Are there space limitations? I would assume from the image that the text can wrap.

**Review by flodolo — COMMENTED** [review]
The rest of the file looks great, thanks for the detailed comments.

File: l10n/templates/strings.properties
Comment (Mardak) [reply l10n/templates/strings.properties]: The text can wrap, and there's plenty of space for long labels:
image

I see that "saves" is used in at least one place on the pocket website: "You are using Pocket without an account. Sign Up to secure your saves and sync them with your phone." Although other parts seem to refer to "saved items" -- https://help.getpocket.com/article/941-where-can-i-find-my-saved-items

I'll check with abenson and pocket people about the preferred wording. From a quick look, it seems that pocket is pretty consistent with "Save to Pocket"

I suppose a localization note of "Saves" being a plural of the result of doing a "Save" or just "Saved Items" and that "Pocket" is the brand…

Comment (flodolo) [reply l10n/templates/strings.properties]: > suppose a localization note of "Saves" being a plural of the result of doing a "Save" or just "Saved Items" and that "Pocket" is the brand…

Yes, that would help and unblock us. The open question is if locales should try to be consistent with Pocket, in case it's a term used in their UI.

Comment (Mardak) [reply l10n/templates/strings.properties]: Looking through the pocket add-on strings, there isn't a "saves" (or a "saved items"):
https://searchfox.org/mozilla-central/source/browser/extensions/pocket/locale/en-US/pocket.properties

Comment (flodolo) [reply l10n/templates/strings.properties]: Sorry, I meant their website UI. The Pocket add-on is, to put it mildly, unmaintained.

And I realized I could just check their website 🐢 It doesn't look like there's anything similar to "Saves" in the UI.

**Comment by Mardak** [comment]
Updated string to "Pocket Saved Items" and the screenshot in the original comment.

**Review by flodolo — APPROVED** [review]

**Comment by Mardak** [comment]
I guess we won't merge until final confirmation from pocket…

**Comment by Mardak** [comment]
Updated to "Pages Saved to Pocket"

**Comment by flodolo** [comment]
@Mardak 
Should we land this?

**Comment by Mardak** [comment]
Yeah if `prefs_highlights_options_pocket_label=Pages Saved to Pocket` is good without a localization note.

**Pull request merged**
